content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -182,10 +184,39 @@ The following Datadog tracing libraries are supported:
 | [.NET][14]                  | [1.18.2][15]                |
 
 
+## Open Telemetry support
+
+RUM also supports several propagator types to connect resources with backends instrumented with Open Telemetry libraries.
+
+{{< tabs >}} {{% tab "Browser RUM" %}}
+1. Setup RUM to connect with APM as described above
+
+2. Modify `allowedTracingUrls` such as:
+    ```javascript
+    import { datadogRum } from '@datadog/browser-rum'
+
+    datadogRum.init({
+        ...otherConfig,
```

Comment:
```suggestion
        # ...otherConfig,
```

Reply:
I kept `...otherConfig` as it is valid JavaScript, and is used in other places in this doc.